### PR TITLE
Marcandea sort content

### PR DESCRIPTION
**Pull request by Marcandea**
I managed to solve the problem with the help of my colleagues and Trainer.

### CODE REVIEW

File: src/test/java/org/fundacionjala/coding/Marcan/SortContentTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        SortContent sortContent = new SortContent();
+        assertEquals("Conidg", sortContent.sortContent("Coding"));
+    }
+
+    /**
+     * This test is for verify to characters inverse inside of the word in a sentence.
+     */
+    @Test
+    public void verifyASentence() {
+        SortContent sortContent = new SortContent();
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/Marcan/SortContentTest.java]: You are repeating this code in both tests
Please put them in @Before method or convert it to a static method.

File: src/test/java/org/fundacionjala/coding/Marcan/SortContentTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.Marcan;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by MarcandeaCabrera on 5/23/2017.
+ */
+public class SortContentTest {
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/Marcan/SortContentTest.java]: I found a bug in your implementation
What should happen in this scenario?

```java
 assertEquals("I am", sortInnerContent.sort("I am"));
```

File: src/main/java/org/fundacionjala/coding/Marcan/SortContent.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    public static final int THREE = 3;
+
+    /**
+     * The following method is for reverse one word.
+     *
+     * @param string The test string is sent.
+     * @return stringBuilder
+     */
+    public String sortContent(String string) {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Marcan/SortContent.java:19]: Try to split in two methods
Now the logic is more complicated

Comment (Marcandea) [reply src/main/java/org/fundacionjala/coding/Marcan/SortContent.java:19]: I guess I should refactor...?

File: src/main/java/org/fundacionjala/coding/Marcan/SortContent.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+     */
+    private String sortInnerContent(String word) {
+        char[] wordInChar = word.substring(1, word.length() - 1).toCharArray();
+        Arrays.sort(wordInChar);
+        StringBuilder str = new StringBuilder();
+        str.append(word.substring(word.length() - 1, word.length()));
+        str.append(wordInChar);
+        str.append(word.substring(0, 1));
+        String reversed = str.reverse().toString();
+        return reversed;
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/Marcan/SortContent.java:52]: try this

```java
return str.reverse().toString();
```

**Pull request closed without merging**